test/controllers/patients_controller_test.rb

```diff
@@ -236,4 +236,116 @@ def teardown; end
       sign_out user
     end
   end
+
+  test 'update status for a patient with no dependents' do
```

Comment:
I added a test for the additional functionality added to `update` as a part of this PR. I wasn't sure if I should add tests for `update_monitoring_fields`, because that function isn't an endpoint that gets hit, it is more of a helper function, so the existing tests do test it indirectly. Let me know if you think that indirect testing via the `update_status` endpoint works, or if you do think we should have direct tests.

Reply:
I think this is fine. Thanks!